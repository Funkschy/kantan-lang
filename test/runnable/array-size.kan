import "io";

def main() {
    io.printf("%ld\n", sizeof *[3]*[4]**Struct);
    io.printf("%ld\n", sizeof [3]*[4]**Struct);
    io.printf("%ld\n", sizeof [2]*[4]**Struct);
    io.printf("%ld\n", sizeof [3][4]**Struct);
    io.printf("%ld\n", sizeof [3][4]Struct);
}

type Struct struct {
    i: i32
}

