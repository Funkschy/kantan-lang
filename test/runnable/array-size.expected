8
24
16
96
48
